db/migrate/20220326155321_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
```

Comment:
- [ ] Let's add which  will create author_id and a corresponding index  (but students can also add column and index in separate commands
`    t.references :author, foreign_key: { to_table: 'users' } )
`